app/modules/general/components/select_dropdown.svelte

```diff
@@ -20,7 +27,8 @@
     e.preventDefault()
   }
 
-  $: currentOption = options.find(option => option.value === value)
+  const optionPlaceholder = { value: 'all', text: I18n('loading') }
```

Comment:
Having all the dropdown menus displaying "loading" (or the even more verbose "chargement en cours") looks a bit unnecessarily bloated; what about just a spinner? I haven't tried it, but it might be quite simple to implement, by having an `{#if}` block within `.inner-select-option`: `{#if option}...{:else}<Spinner />{/if}`

Reply:
I dont think clicking on several/all facets selectors while loading is a reasonable usecase, why would one click on everything after they know one facet is loading? (besides trolling/testing, which can/must be ignored). And even if they do, then "bloating" a bit (and temporarily) the interface looks relevant to me, interface is shouting  "wait, please!", more and more, as more facets selectors are displayed.

Besides this example, i will also answer on the argument itself:
When facets are loading, selector input is already displaying a text stating "Loading", as a first attempt to inform that one must wait. I can see 2 reasons why the user would still choose to click anyway: 
- not reading the "Loading" text (ie. knowing the interface, clicking by habit),
- still wanting to explore whats happening, as an attempt to understand why this loading state is so long (and it can really take a solid minute for 1000 entities cf. subjects layout in #383).
I think your suggestion (having spinner without help text) would satisfy the first case above, but if one would like to know why, then inventaire can display a hint (aka. how heavy the entities amount is `urisSize`).

If you are not convinced by this argumentation, it's fine for me to delete this text, as it simplifies the code, but i prioritize "displaying information when available" over "code simplification".

Reply:
Are we talking of the same thing? This is what I have in mind and find a bit too verbose:
![Capture d’écran du 2023-02-28 20-38-25](https://user-images.githubusercontent.com/1596934/221960779-3bec7538-06fa-4bba-adc9-39715fc66fc7.png)

Reply:
> it can really take a solid minute for 1000 entities

Yes, those layouts aren't doing pagination anymore, which might be an issue for publishers in particular. I was considering the possibility to move facets to the server, just like `inventory-view`, but with better caching, which would mean having to properly design the cache invalidation... That's why I would be in favor of not making those layouts pixel perfect just yet, there might still be some heavy work required.

Reply:
> Are we talking of the same thing?

yes, I misunderstood you here, this it not the behavior i had in mind. Could be because of the rebase? Anyway. What i had in mind was to fakingly display "All", and then if a user is quick enough to click and display the dropdown, then display the loading message with the spinner.

This is done with bce58dec1 but its not what you suggested, would that be okay for you?